packages/ra-core/src/controller/field/getResourceLinkPath.ts

```diff
@@ -62,17 +62,19 @@ const getResourceLinkPath = ({
         );
     }
     const sourceId = get(record, source);
-    const rootPath = basePath
-        ? basePath.replace(resource, reference)
-        : `/${reference}`;
+    const rootPath = basePath ?
+        basePath.replace(resource, reference) :
+        `/${reference}`;
     const linkTo: LinkToType = linkType !== undefined ? linkType : link;
 
+    // @rattrayalex: below it's kinda weird to do this, should we put it in parens?
+
     // Backward compatibility: keep linkType but with warning
-    return !linkTo
-        ? false
-        : typeof linkTo === 'function'
-        ? linkTo(record, reference)
-        : linkToRecord(rootPath, sourceId, linkTo as string);
+    return !linkTo ?
+        false :
+    typeof linkTo === 'function' ?
+        linkTo(record, reference) :
+        linkToRecord(rootPath, sourceId, linkTo as string);
```

Comment:
Weird edge case: nested ternaries in a return statement look weird. Maybe use parens?

Reply:
Yeah, the indent is off.